core/common-util/src/main/java/uk/gov/gchq/gaffer/commonutil/OneOrMore.java

```diff
@@ -117,14 +117,10 @@ public void removeAnyItem() {
     }
 
     public int size() {
-        if (null == collection) {
-            if (null != singleItem) {
-                return 1;
-            }
-            return 0;
+        if (null != collection) {
```

Comment:
bit nit picky but probably should be `collection != null` for consistency and easier reading